README.md

```diff
@@ -1 +1,5 @@
-# todo-list
```

Comment:
Please describe this only steps to start development. 
e.g. 
# TO-DO List

---
To star development first of all install all needed dependences. To do it open the terminal and run the command ```npm install``` 

To start the dev-server please run next command ```npm run start```

Reply:
Done